### ISSUE TITLE
convert datetime handling from strings to unix-time based internal variable

### ISSUE THREAD

**Issue by andywood** [NOAA-OWP/noah-mp-modular#16]
current date/time handling is based on strings, which is brittle and prevents using most fortran time utilties and conventions

**Comment by SnowHydrology** [comment]
Agreed that this would be useful, but let's chart out a strategy that prioritizes the following:

- [ ] Changes to one module are easily portable to another module
- [ ] Noah-MP-Modular enhancements do not duplicate Nextgen functionality

**Comment by SnowHydrology** [comment]
@andywood, do your commits from [PR 30](https://github.com/NOAA-OWP/noah-mp-modular/pull/30) close this issue?

**Comment by andywood** [comment]
Not quite - there are a lot of string-based operations still in the ATM
module, I think -- these would be much simpler after switching to unix
times.

**Comment by SnowHydrology** [comment]
That makes sense. I'll leave this open for now. Thanks!
